docs/buckets.rst

```diff
@@ -0,0 +1,62 @@
+Buckets
+#######
+
+Buckets are a way to group users around some data.
+
+Basically has got an and a list of principals that can administrate
+it.
+
+A bucket can be created using a PUT on the bucket URI:
+
+.. code-block:: http
+
+   > PUT /buckets/servicedenuages HTTP/1.1
+   < 201 Created
+
+    {
+      "id": "servicedenuages",
+      "principals": ["email:natim@example.com"]
```

Comment:
Having "principals" returned here doesn't make a lot of sense to me, we should probably have a list of permissions and eventually a group "owners"